docs/config/misc.md

```diff
@@ -65,6 +65,32 @@ return {
   -- Set this to "NeverPrompt" if you don't like confirming closing
   -- windows every time
   window_close_confirmation = "AlwaysPrompt",
+
+  -- When true (the default), watch the config file and reload it
+  -- automatically when it is detected as changing.
+  -- When false, you will need to manually trigger a config reload.
+  automatically_reload_config = true,
+
+  -- The github releases section is checked regularly for updated stable
+  -- releases and show a simple UI to let you know about the update,
+  -- with links to download/install it. It doesn't automatically download
+  -- the release. No data are collected for the wezterm project as part
+  -- of this.
+  -- By default it is checked once every 24 hours.
+  -- Set `check_for_updates` to `false` to disable this completely or set
+  -- `check_for_updates_interval_seconds` for an alternative update interval.
+  -- You can also disable the update window with `show_update_window`.
+  check_for_updates = true,
+  check_for_updates_interval_seconds = 86400,
+  show_update_window = true,
```

Comment:
For these 3, do I make a single file for all of them? (like update.md somewhere) or make a file for each? (duplicating some info)

Reply:
If it's clearer to duplicate (perhaps it's just a couple of lines in two places) then I'd lean towards that, but otherwise, I'd suggest putting the main explanation into one of them and having the other related options briefly describe their purpose and link to the more fleshed out information.

Reply:
done!